README.md

```diff
@@ -10,7 +10,7 @@
 
 To install the Booking Product Extension, follow these steps:
 
-##### 1. Unzip the respective extension zip and then merge "packages" and "storage" folders into project root directory.
+##### 1. Unzip the respective extension zip and then merge "packages/Webkul" folders into project root directory.
```

Comment:
@amit-webkul 

Here only **packages** will appear.

Note - Correct format for booking product folder managemenr is we will create **packages** folder and inside that **Webkul** folder and in Webkul folder booking product package folder.

Reply:
There is no need to do this.